### PR TITLE
Add copy updates

### PR DESCRIPTION
**Pull request by chrisarusso**
* Link phone, and map in contact page
 * Update copy on open source page to be more informative
 * Update logic in services ind pages presentation
 * Update CTA in services page
 * Update Elevate Your Org CTA
 * Update services landing and ind page copy

### CODE REVIEW

**Comment by chrisarusso** [comment]
@AnneTee - I'm going to merge so that this is live immediately, however, please do retroactively review. I did not run through grammar or spell-checking.

File: _services/design-ux.md [hunk 1]
```diff
@@ -4,13 +4,14 @@ title: Design / UX
 slug: design-ux
 weight: 3
 summary: |
-  Creating a beautiful design is a start. To design with purpose, aware of the user's every hope and dream, focused on usability that empowers content editors to soar: now that's another thing. UX in every permutation is what drives our design choices.
+  We design with purpose, guided by organizational goals and an acute awareness for all interaction stakeholders. We empower site admins to be more productive while crafting a beautiful, pleasant user journey.
```
Comment (AnneTee) [inline _services/design-ux.md:7]: @chrisarusso 

> an acute awareness for all interaction stakeholders

This is unclear to me...what is an interaction stakeholder?

File: _services/design-ux.md [hunk 2]
```diff
@@ -4,13 +4,14 @@ title: Design / UX
...
-services: [User Experience Strategy, Wireframing & Prototyping, Interface Design, Conversion Optimization, Concepting & Ideation, E-Commerce Web Design, Styleguide Design, Usability & User Testing]
+services: [User Experience Strategy, Concepting & Ideation, Wireframing & Prototyping, Style Guide Design, Conversion Optimization, Usability & User Testing]
 
 ---
 
-Through conceptualizing, UX design, and iterative wireframing, we design with mindfulness relevant to our clients' audiences and goals. We provide flexible, maintainable, and responsive custom designs using the best of the myriad web frameworks to our advantage, enabling us to prototype and build quickly. When it is the right fit, we partner with a design agency with whom we have an established relationship.
\ No newline at end of file
+Through rapid prototyping, leveraging the best of the modern web, our design process is iterative and translates efficiently to the final product. Driven by developing a deep understanding of the goals of our partners and their audiences, we craft flexible, usable, future-considerate responsive designs that empower content editors, are fit for all devices, and are a pleasure to use.
```
Comment (AnneTee) [inline _services/design-ux.md:16]: I'd change to:

> Through rapid prototyping with the best modern tools, our design process...

This isn't just to remove "leveraging," I promise - I think "the best of the modern web" needs to be more specific and the phrasing needs some work.

Comment (AnneTee) [reply _services/design-ux.md:16]: I'd also rephrase the second sentence and change "future-considerate" to "future-proof":

We develop a deep understanding of the goals of our partners and their audiences in order to craft flexible, usable, future-proof responsive designs that empower content editors, are fit for all devices, and are a pleasure to use.

File: _services/development.md [hunk 3]
```diff
@@ -4,15 +4,15 @@ title: Development
 slug: development
 weight: 2
 summary: |
-  Turning real-world challenges into technical solutions is among our favorite activities amidst the beakers and bunsen burners in the Savas lab. Developing novel functionality that is tailor-made for each client's needs drives our creative spirit.
+  Manifesting strategic organizational challenges via technical execution is one of the most cherished activities we participate in. We employ a wide variety of tools to tailor-make solutions for our partner's constantly evolving needs.
```
Comment (AnneTee) [inline _services/development.md:7]: The first sentence is pretty hard to digest and I'd recommend not ending the sentence in a preposition. I don't think that's a hard and fast rule in general writing but I'd avoid it in short, hard-hitting sections of the site like this - I think it weakens the statement.

I'd also avoid using "tailor-make;" use "tailor-made" if you can but if not change "to make custom solutions."

I'd also change "partner's" to "partners'"

File: _services/strategy.md [hunk 4]
```diff
@@ -4,15 +4,13 @@ title: Strategy / Consulting
 slug: strategy
 weight: 1
 summary: |
-  Simple execution is never simple until all avenues have been explored. A stitch in time saves nine, and a minute of planning saves 10 in execution. We've learned these lessons and cherish a challenge to think creatively for the most appropriate solutions for our partners.
+  Our seasoned team combines wisdom from years of digital consulting experience with a holistic eye to think creatively, leaving no stone unturned to discern the best solutions for our partners.
```
Comment (AnneTee) [inline _services/strategy.md:7]: Pairing up something really strong like "wisdom from years of digital consulting experience" with "a holistic eye to think creatively" isn't working...I think because eyes don't think creatively, and because you're comparing our collective wisdom with a single thing that we can't share (eye). I'd change to:

Our seasoned team combines wisdom from years of digital consulting experience with a holistic process, leaving no stone unturned....

File: _services/strategy.md [hunk 5]
```diff
@@ -4,15 +4,13 @@ title: Strategy / Consulting
...
+services: [Content Strategy, SEO, Technical Audit / Planning, Channel Identification, Competitive Analysis, Analytics Insights, Information Architecture, User Journey Mapping, Brand Assessment, Digital Marketing, Campaign Planning  ]
 
 ---
 
-We empower organizations to sift through seemingly limitless technical choices and focus on what the organization is trying to achieve. Whether through a technical audit, SEO, content analysis, or other measures, we ensure our clients use their resources most effectively to meet their needs and reach their audiences.
-
-We lean on lessons learned to inform our process while catering to each organization's unique needs in space and time.
+We turn organizational data into actionable information so our partners can focus on what they know best: their organization's needs. We always listen carefully to our client's pain points and their visions of a better future, while providing a complementary, fresh perspective viewed from outside of the institutional lens. Some of the highest impact we can provide is to help steer the planning and technical investment methodically in concert with our partners to pursue the best opportunities available.
```
Comment (AnneTee) [inline _services/strategy.md:16]: I'm torn over whether we should speak of clients in the plural sense (so instead of "client's" you'd want "clients'") or not. We're generally speaking of them in the plural so I'd recommend maintaining that plural throughout. This will mostly mean updating possessives like "organizations'" and "clients'"

> Some of the highest impact we can provide is to...

This sounds weak; I'd rephrase as:

> We can provide high-impact changes by steering the planning and technical investment....

File: _services/support.md [hunk 6]
```diff
@@ -4,13 +4,13 @@ title: Support / Maintenance
 slug: support-maintenance
 weight: 4
 summary: |
-  Maintenance work doesn't typically evoke dreamy visions like building a new application does. However, we recognize the original investment is only truly borne when it's in reliable operation. With the help of generous robots (who we pay), and partners, we ensure 24/7/365 support.
+  An application serves its purpose best when running. With the help of robots, we ensure maximum uptime and provide ongoing iterative improvements that a modern web system requires to remain relevant.
```
Comment (AnneTee) [inline _services/support.md:7]: Change last part to "that modern web systems require to remain relevant."

File: _services/support.md [hunk 7]
```diff
@@ -4,13 +4,13 @@ title: Support / Maintenance
...
+services: [Uptime monitoring, 24x7x365 Support, Bug Fixing,  Feature Enhancements, Drupal Security Updates, Disaster Recovery Planning, Backup Strategy]
 
 ---
 
-A site with an audience that is engaged on a daily basis requires the same technical attention behind the scenes. We provide just that. Our robot friends ensure that our clients' sites are alive and well with robust monitoring. Our humans quickly test and apply security updates and make recommendations on sustainability best practices. A support & maintenance agreement will often start with an audit.
\ No newline at end of file
+An organization with a captive audience who cares deeply about its missions requires the same technical attention behind the scenes to ensure its applications are functional. We provide this assurance through best-in-class monitoring tools, around-the-clock application support engineers, and a genuine yearning to see our partners and their applications thrive. With a broad and deep understanding of security risks in the ever-interconnected world, we consult on all aspects from the technical stack to organizational stakeholders to maximize the effectiveness of our partner's web systems.
```
Comment (AnneTee) [inline _services/support.md:16]: > An organization with a captive audience who cares deeply about its missions

It's unclear whether the group caring deeply about its mission is the organization or the audience. 

> genuine yearning

I'd nix "yearning," sounds awkward

> broad and deep

I  know they mean different things but it seems redundant. I'd opt for "deep."

> partner's

Change to partners'

**Review by AnneTee — COMMENTED** [review]
@chrisarusso I left you a lot of comments about phrasing. A lot of the paragraphs are difficult to digest to me and I'm getting lost in the complicated and sometimes awkward phrasing. I checked out a few others' websites and think we should try to stick to slightly simpler phrasing that emphasizes clearly what we do. I liked [this example](https://www.fourkitchens.com/services/technology/)

I don't know what your exact goals are with voice but these are my thoughts...
